<launch>
	<arg name="psbmpc_config" default="psbmpc.yaml"/>
	<arg name="psbmpc_waypoints" default=""/>

	<!-- Launch PSBMPC visualization node-->
	<node name="psbmpc_visualization" pkg="psbmpc" type="psbmpc_visualization_node" ns="psbmpc" respawn="false" output="screen" launch-prefix="" >
		<rosparam file = "$(find psbmpc)/config/$(arg psbmpc_config)" command = "load" />
		<rosparam file = "$(find psbmpc)/config/psbmpc_visualization.yaml" command = "load" />
	</node>
</launch>
